app/views/courses/_walkout_button.html.erb

```diff
@@ -0,0 +1,5 @@
+<% if signed_in? && current_user.student? %>
+  <%= link_to @student, method: :delete, data: { confirm: "Вы уверены, что хотите покинуть курс?" } do %>
+    <span class="btn btn-danger">Отписаться</span>
```

Comment:
This will remove some statistics about the term which doesn't sound good.
And what if a student then decides to enroll back?